### ISSUE TITLE
[Firefox] Don't show rules for connections that will be blocked

### ISSUE THREAD

**Issue by Mosrite** [EFForg/https-everywhere#1889]
Many users use addons to block ads, tracking etc. Thereby, not only single elements, but often entire third-party domains will be blocked. On Firefox HTTPS Everywhere indicates rules for such domains as active even if there won't be an actual connection. This goes especially for Google or Facebook resources that will be blocked by many anti-tracking addons.
See also:
https://github.com/chrisaljoudi/uBlock/issues/1408
https://github.com/gorhill/uBlock/issues/245
I'm aware that this is mostly a cosmetic issues, but I think, it would be nice, if HTTPS Everywhere would only indicate rules as active which belong to connections that will actually be made. Plus, maybe it will even save some resources if HTTPS Everywhere wouldn't have to care about requests that won't lead to a connection after all.

**Comment by semenko** [comment]
Closing as wontfix -- though feel free to send a pull request if you'd like to try to implement this.

As is marked on those uBlock threads, there's a race in Chrome between uBlock and HTTPSe for URL rewrites and blocking -- making this unstable at best.
